cmake/TileDB-Superbuild.cmake

```diff
@@ -128,16 +128,31 @@ endif()
 # Set up the regular build (i.e. non-superbuild).
 ############################################################
 
-ExternalProject_Add(tiledb
-  SOURCE_DIR ${PROJECT_SOURCE_DIR}
-  CMAKE_ARGS
-    -DTILEDB_SUPERBUILD=OFF
-    ${INHERITED_CMAKE_ARGS}
-    ${FORWARD_EP_CMAKE_ARGS}
-  INSTALL_COMMAND ""
-  BINARY_DIR ${CMAKE_CURRENT_BINARY_DIR}/tiledb
-  DEPENDS ${TILEDB_EXTERNAL_PROJECTS}
-)
+if(WIN32)
+  ExternalProject_Add(tiledb
+    SOURCE_DIR ${PROJECT_SOURCE_DIR}
+    CMAKE_ARGS
+      -DTILEDB_SUPERBUILD=OFF
+      ${INHERITED_CMAKE_ARGS}
+      ${CMAKE_CXX_FLAGS}
```

Comment:
CMAKE_CXX_FLAGS is included in INHERITED_CMAKE_ARGS

Reply:
Removed